internal/iam/repository_test.go

```diff
@@ -160,6 +160,7 @@ func Test_Repository_delete(t *testing.T) {
 		err = db.TestVerifyOplog(t, rw, s.PublicId, db.WithOperation(oplog.OpType_OP_TYPE_DELETE), db.WithCreateNotBefore(5*time.Second))
 		require.NoError(err)
 	})
+	oplog.TestOplogDeleteAllEntries(t, db.New(conn).UnderlyingDB())
 	kms.TestKmsDeleteAllKeys(t, conn)
```

Comment:
There seems to be a tight coupling between these two functions. `oplog.TestOplogDeleteAllEntries` is always called right before `kms.TestKmsDeleteAllKeys`. I commented out the call to `oplog.TestOplogDeleteAllEntries` which caused an error in `kms.TestKmsDeleteAllKeys`. Given this coupling, should we just call `oplog.TestOplogDeleteAllEntries` from `kms.TestKmsDeleteAllKeys`?

Reply:
That's a great idea